Automattic/studio
# Update dependabot dependencies

Related to p1714650498693029-slack-C06DRMD6VPZ

## Proposed Changes

- Update dependencies suggested by [dependabot](https://github.com/Automattic/studio/security/dependabot) except lodash

## Testing Instructions



- Run `npm install` to use the new dependnecies
- Run `npm start` to start Studio
- Make an holistic test in all the features, creating a site, creating a demo site.
- Observe the app runs correclty

## Pre-merge Checklist
